thf-conference-api/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "prestart": "npm install",
-    "start": "nodemon index.js"
+    "start": "node index.js"
```

Comment:
Sobe um script "dev" (sugestão) com o nodemon para quando for necessário rodar em desenvolvimento.
Ex: "dev": "nodemon index.js"